src/components/FormHelpMessage.tsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import {View, ViewStyle, StyleProp} from 'react-native';
+import isEmpty from 'lodash/isEmpty';
+import Icon from './Icon';
+import * as Expensicons from './Icon/Expensicons';
+import Text from './Text';
+import themeColors from '../styles/themes/default';
+import styles from '../styles/styles';
+import * as Localize from '../libs/Localize';
+
+type FormHelpMessageProps = {
+    /** Error or hint text. Ignored when children is not empty */
+    message?: string | Array<string | Record<string, string>>;
+
+    /** Children to render next to dot indicator */
+    children?: React.ReactNode;
+
+    /** Indicates whether to show error or hint */
+    isError?: boolean;
+
+    /** Container style props */
+    style?: Array<StyleProp<ViewStyle>>;
+};
+
+function FormHelpMessage({message = '', children, isError = true, style = []}: FormHelpMessageProps) {
+    if (isEmpty(message) && isEmpty(children)) {
```

Comment:
is possible to use native methods here?

Reply:
In this case better to use `isEmpty` since here we need a complex check

Reply:
I'm only concerned about this `isEmpty()` check on children which can get costly and it runs inside if the render function.

Reply:
I agree with @fabioh8010 we should consider using native methods instead, let's try to get rid of lodash 😄

Reply:
In [ReportUtils PR](https://github.com/Expensify/App/pull/29012) I added a function for checking if the object is empty, you can copy it here and get rid of lodash isEmpty

Reply:
Faced with a complex case of this replacement

After discussion with @fabioh8010 we decided to keep lodash here